frontend/packages/kubevirt-plugin/src/kubevirt-dependencies/components/FilteredSelect/FilteredSelect.tsx

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable array-callback-return, consistent-return */
+import * as React from 'react';
+import {
+  Select as SelectDeprecated,
```

Comment:
Any deprecated components need to be removed and replaced with the PF5 equivalent; otherwise restoring this PR as is will block the removal of PF4.